docs/articles/expensify-classic/getting-started/Invite-Members.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Invite Members
+description: Learn how add your employees to submit expenses in Expensify
+---
+# Overview
+
+To invite your employees in Expensify, simply add them as members to your Workspace.
+
+# How to Invite members to Expensify
+
+## Inviting Members Manually
+
+Navigate to **Settings > Workspace > Group > *Workspace Name* > People** - then click Invite and enter the invitee's email address.
+
+Indicate whether you want them to be an Employee, Admin, or Auditor on the Workspace.
+
+If you are utilizing the Advanced Approval feature, you can specify who members should submit their expense reports to and who an approver should send approved reports to for the next step in the approval process. If someone is the final approver, you can leave this field blank.
```

Comment:
"and who an approver should send approved reports to for the next step in the approval process" sounds off @NicMendonca, I think this still needs to be fixed?

Reply:
I left the same comment previously [here](https://github.com/Expensify/App/pull/28276#pullrequestreview-1652089064), something is off with the copy. Should it say "you can specify to whom members should submit their expense reports"?

Reply:
@pecanoro but that's not the feature. In the Advanced approval feature you a) specify who members should submit their expense reports to  - this is already in the copy) and then b) who *that* approver, approvers and forwards their reports to for final approver

Reply:
Hmm reading it over again it actually sounds fine to me, maybe I read it out of context and it sounded off, sorry!

Reply:
I still believe there is something wrong with that sentence and it should be rephrased it since it's really hard to understand 😅

Reply:
I agree that this sentence is long and difficult to read. I've added a suggestion on how to rephrase it below.